src/scripts/metadata_update.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+# .. See the NOTICE file distributed with this work for additional information
+#    regarding copyright ownership.
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#        http://www.apache.org/licenses/LICENSE-2.0
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import argparse
+import re
+from sqlalchemy.engine.url import make_url
+from ensembl.production.metadata.updater import CoreMetaUpdater
+def main():
+    parser = argparse.ArgumentParser(description='Method for loading data from a spcific database to a metadata registry')
+
+    parser.add_argument('-m', '--uri', help='Metadata database URI', required=True)
+    parser.add_argument('-d', '--database_uri', help='URI of database to load', required=True)
+    parser.add_argument('-r', '--release', help='Release')
```

Comment:
Not sure we need to set this, is it meant to update an already released genome data? I think we said that the release processing would be independent from handover, so we'll only create the dataset and assign release later in the processing.

Reply:
I was actually talking about the `--release` param, the others are fine. Assuming those are "r/w" connexion strings, right?

Reply:
We'll keep it for the future then.